confgenerator/confmerger.go

```diff
@@ -216,13 +192,12 @@ func mergeConfigs(original, overrides *UnifiedConfig) {
 		for k, v := range overrides.Logging.Processors {
 			original.Logging.Processors[k] = v
 		}
-
-		// Overrides logging.exporters.
-		for k, v := range overrides.Logging.Exporters {
-			original.Logging.Exporters[k] = v
-		}
+		// Skip deprecated logging.exporters.
+		// Override logging.service.pipelines
 		if overrides.Logging.Service != nil {
 			for name, pipeline := range overrides.Logging.Service.Pipelines {
+				// skips logging.service.pipelines.*.exporters
+				pipeline.ExporterIDs = nil
```

Comment:
Do we need this line after the exporter overrides is removed below?

Reply:
Ah, answered myself: we need it for non-default pipelines. Ignore the comment above